project/settings/local.py

```diff
@@ -37,6 +40,3 @@
         },
```

Comment:
Bi moralo tudi to ven?

Reply:
Ja, verjetno res. @jureslak to si dal ti noter, kam spada?

Reply:
To je bilo noter, ko sem vzpostavljal LDAP in sem lokalno debuggal LDAP nastavitve in povezljivost.

Obcasno je fino imeti tudi lokalno LDAP postavljen, ko se dela kaj na tem, ampak verjetno je v splosnem bolj uporabno to takrat narediti na roke, glede na to da se vecinoma ne spreminja in da rabis bit povezan na fmf omrezje, da lahko razvijas. Tako da gre res lahko ven.

Reply:
A lahko vržemo ven kar celoten `LOGGING`?

Reply:
Ja, lahko. Unrelated issue je, da bi ga bilo verjetno fino imet na serverju, ampak za lokalni razvoj pa ni potreben.